src/Test/Tasty/Bench.hs

```diff
@@ -740,6 +754,11 @@ data Timeout
 newtype RelStDev = RelStDev Double
   deriving (Show, Read, Typeable)
 
+-- | Whether to measure CPU time or wall-clock time. See the module
+-- header or the README for a more detailed discussion.
+data TimeMode = CpuTime | WallTime
```

Comment:
Note that there is no way to use `WallTime` when `tasty` flag is disabled. I suggest defining
```suggestion
data TimeMode = CpuTime 
#ifdef MIN_VERSION_tasty
  | WallTime
#endif
```
and similar guard in `getTimePicoSecs`. In this case you can rely on `Test.Tasty.Runners.getTime` instead of  `getMonotonicTimeNSec` or `Clock.getTime`.

Reply:
Will do :+1:, even though we will lose some precision as we have to go through `Double` in contrast to the existing mechanism. 

Also, if we ever want to expose sth like `measureWallClockTime` for monotonic time measurement without tasty, we will have to think about how to measure it, also see your [comment above](https://github.com/Bodigrim/tasty-bench/pull/33#discussion_r878770129):

> let's not expose measureWallClockTime **yet**.

Reply:
There are very few users, who disable `tasty` flag; mostly GHC developers, and I doubt they ever need wall clock time for their investigations.

Reply:
Hmm, I am only just noticing that tasty started to export `getTime` in version 1.2.2 ([changelog](https://hackage.haskell.org/package/tasty-1.4.2.3/changelog)), any thoughts?

Reply:
See https://github.com/amesgen/tasty-bench/actions/runs/2366523959 for the failing CI jobs.

Reply:
Feel free to use `#if defined(MIN_VERSION_tasty) && MIN_VERSION_tasty(1,2,2)` instead of `#ifdef MIN_VERSION_tasty`, not a big deal.

Reply:
`#if defined(MIN_VERSION_tasty) && MIN_VERSION_tasty(1,2,2)` seems to error with

```
     error: missing binary operator before token "("
     #if defined(MIN_VERSION_tasty) && MIN_VERSION_tasty(1,2,2)
```

when `tasty: False` for some reason, so I went with sth slightly more verbose.